### PR TITLE
feat: deprecate goerli

### PR DESCRIPTION
**Pull request by sanyu1225**
## Related Links

**Asana**:  https://app.asana.com/0/1203318877558072/1206733445446676/f

**Mockup**:  

## Checklist

- [X] Pasted Asana or Mockup link.
- [X] Tagged labels.

### CODE REVIEW

File: src/contracts/index.ts [hunk 1]
```diff
@@ -13,10 +13,6 @@ export const ContractInfos: ContractInfosType = {
     abi: valueDappAbi,
     address: "0x009c403BdFaE357d82AAef2262a163287c30B739",
   },
-  [EvmChainId.EthereumGoerli]: {
```
Comment (mordochi) [inline src/contracts/index.ts:16]: 這邊也有對應到一些其他鏈 goerli testnet 的合約 也可以一起拿掉

File: src/types/ChainTypes.ts [hunk 2]
```diff
@@ -15,7 +15,6 @@ export enum EvmChain {
 
 export enum EvmChainId {
   Ethereum = "0x1",
-  EthereumGoerli = "0x5",
```
Comment (mordochi) [inline src/types/ChainTypes.ts:18]: 這邊還有對應到一些其他鏈 goerli testnet 的 chain id

**Review by mordochi — APPROVED** [review]

File: src/types/ChainTypes.ts (hunk 2 above)
Comment (sanyu1225) [reply src/types/ChainTypes.ts:18]: [5e3b9e6](https://github.com/blocto/sdk-toolkit/pull/102/commits/5e3b9e608ae78625689e93254f502e8445e64a16)

File: src/contracts/index.ts (hunk 1 above)
Comment (sanyu1225) [reply src/contracts/index.ts:16]: [5e3b9e6](https://github.com/blocto/sdk-toolkit/pull/102/commits/5e3b9e608ae78625689e93254f502e8445e64a16)

**Pull request merged**